README.md

```diff
@@ -1,23 +1,29 @@
-# Appcelerator Titanium Package for Atom [![Build Status](https://travis-ci.org/appcelerator/atom-appcelerator-titanium.svg?branch=master)](https://travis-ci.org/appcelerator/atom-appcelerator-titanium)
+# Titanium SDK Package for Pulsar [![Build Status](https://travis-ci.org/appcelerator/atom-appcelerator-titanium.svg?branch=master)](https://travis-ci.org/appcelerator/atom-appcelerator-titanium)
 
-Appcelerator Titanium build tools and UI for the [Atom text editor](https://atom.io).
+Titanium SDK build tools and UI for the [Pulsar text editor](https://pulsar-edit.dev/) (formally Atom).
 
-<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Appcelerator Titanium for Atom screenshot" />
+<img src="https://user-images.githubusercontent.com/10667698/42400977-8e552cde-8141-11e8-8696-1380eb9dd581.png" width="900px" alt="Titanium SDK for Pulsar screenshot" />
 
 ## Getting Started
 
-* Atom 1.21 (or later) is required.
-* Some features are implemented as providers for services in the [Atom IDE](https://ide.atom.io) package. Install `atom-ide-ui` package.
-* Install `appcelerator-titanium` package.
+* Pulsar is required.
+* Install `pulsar-titanium` package.
 
-### Using the Atom package manager
+### Using the Pulsar package manager
 
-Preferences/Settings > Install > Search for `appcelerator-titanium`.
+Preferences/Settings > Install > Search for `pulsar-titanium`.
 
 ### Using the CLI:
 
+Use `ppm` (Pulsar package manager) or `apm` (Atom package manager):
+
+```
+ppm install pulsar-titanium
+```
+
+You can also use the repo URL to install the latest repo version:
 ```
-apm install appcelerator-titanium
+ppm install https://github.com/tidev/pulsar-titanium
```

Comment:
I think they're leaning towards `pulsar --package` now over `ppm` (and I've had issues with it on Windows)
```suggestion
pulsar --package install https://github.com/tidev/pulsar-titanium
```